From linux-usb-devel-bounces@lists.sourceforge.net Mon Mar 12 01:08:54 2007
From: "Engelmayer Christian" <Christian.Engelmayer@frequentis.com>
Date: Mon, 12 Mar 2007 09:08:36 +0100
Message-ID: <9EE81EBECB70824985BE0EDA8DC9B5158C33CC@VIECLEX01.frequentis.frq>
From: "Engelmayer Christian" <Christian.Engelmayer@frequentis.com>
To: <greg@kroah.com>
Subject: ehci-fsl: change SI_CTRL, PRI_CTRL register offsets according to errata
Content-Type: text/plain; charset="us-ascii"

From: Christian Engelmayer <Christian.Engelmayer@frequentis.com>

Correct the offsets of the SI_CTRL, PRI_CTRL registers according to
the Reference Manual errata sheet in order to prevent unwanted
settings regarding burst transactions and priority states.

Signed-off-by: Christian Engelmayer <Christian.Engelmayer@frequentis.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/ehci-fsl.h |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/drivers/usb/host/ehci-fsl.h
+++ b/drivers/usb/host/ehci-fsl.h
@@ -31,7 +31,7 @@
 #define FSL_SOC_USB_SNOOP1	0x400	/* NOTE: big-endian */
 #define FSL_SOC_USB_SNOOP2	0x404	/* NOTE: big-endian */
 #define FSL_SOC_USB_AGECNTTHRSH	0x408	/* NOTE: big-endian */
-#define FSL_SOC_USB_SICTRL	0x40c	/* NOTE: big-endian */
-#define FSL_SOC_USB_PRICTRL	0x410	/* NOTE: big-endian */
+#define FSL_SOC_USB_PRICTRL	0x40c	/* NOTE: big-endian */
+#define FSL_SOC_USB_SICTRL	0x410	/* NOTE: big-endian */
 #define FSL_SOC_USB_CTRL	0x500	/* NOTE: big-endian */
 #endif				/* _EHCI_FSL_H */
